iot_config/migrations/20241203193310_add_organization_locks.sql

```diff
@@ -0,0 +1,23 @@
+create table organization_locks (
+    organization varchar(255) primary key not null,
+    locked bool default false,
+
+    inserted_at timestamptz not null default now(),
+    updated_at timestamptz not null default now()
+);
+
+select trigger_updated_at('organization_locks');
```

Comment:
```suggestion
    organization VARCHAR(255) PRIMARY KEY NOT NULL,
    locked BOOL DEFAULT false ,
    inserted_at TIMESTAMPTZ NOT NULL DEFAULT now(),
    updated_at TIMESTAMPTZ NOT NULL DEFAULT now()
);

SELECT trigger_updated_at('organization_locks');
```

Reply:
for that matter, why not make the `organization TEXT PRIMARY KEY NOT NULL` to be in line with the rest of the organization records in the testing/postgres-sink table records?

Reply:
We want organizations to initialize unlocked?